packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -267,12 +267,13 @@ export class QuoteRequestor {
     }
 
     private _makerSupportsPair(makerUrl: string, makerAssetData: string, takerAssetData: string): boolean {
-        const makerTokenAddress = getTokenAddressOrThrow(makerAssetData);
```

Comment:
Could this be easier if:
1. `getTokenAddressOrThrow` returns a lowercase address
2. you reduce the amount of `toLowerCase()` calls by creating 2 interim variables

?